src/desktop/apps/article/routes.js

```diff
@@ -224,7 +228,10 @@ export function classic(req, res, next) {
 }
 
 export function amp(req, res, next) {
-  const article = new Article({ id: req.params.slug })
+  const slug = req.url.split("/")[2]
+  const article = new Article({
+    id: slug,
```

Comment:
Since `id` and `slug` are being assigned to one another (yet are different terminology-wise across artsy systems) might be worth a tiny comment mentioning the bug / mixup